### ISSUE TITLE
Wrong indentation after double single-statement for loop

### ISSUE THREAD

**Issue by GoogleCodeExporter** [google/closure-linter#58]
```
What steps will reproduce the problem?
1. gjslint --nojsdoc --strict lint.js

What is the expected output? What do you see instead?
Expected no errors, but got
Line 6, E:0006: Wrong indentation: expected any of {4, 14} but got 2

What version of the product are you using? On what operating system?
Closure linter 2.3.7 with python 2.7 on Ubuntu 12.04.1

Please provide any additional information below.

```

Original issue reported on code.google.com by `andersen...@gmail.com` on 26 Nov 2012 at 11:53

Attachments:
- [lint.js](https://storage.googleapis.com/google-code-attachments/closure-linter/issue-58/comment-0/lint.js)
